### ISSUE TITLE
dials-v3-14-1-linux-x86_64.tar.xz does not install on Silicon Macs

### ISSUE THREAD

**Issue by lucajovine** [dials/dials#2400]
On a M2 Mac running Ventura 13.3.1:

```
> install --prefix=/usr/local/dials/
Traceback (most recent call last):
  File "/usr/local/dials/dials-installer/./bin/install.py", line 311, in <module>
    installer(sys.argv[1:]).install()
  File "/usr/local/dials/dials-installer/lib/libtbx/auto_build/install_distribution.py", line 188, in install
    self.basic_setup()
  File "/usr/local/dials/dials-installer/lib/libtbx/auto_build/install_distribution.py", line 219, in basic_setup
    raise InstallerError("""
InstallerError:
  mac-intel-osx-x86_64-10.9 is not a supported platform, installation aborted
    use the --try-unsupported option to attempt installation on this platform
    use the --no-gui option for a core DIALS installation
```

Are Silicon Macs truly still unsupported?

**Comment by lucajovine** [comment]
And actually... the above suggestion does not work:

```
 ./install --try-unsupported --prefix=/usr/local/dials/

    ==========================================================================
                          DIALS Installation

                          version: v3-14-1
                     machine type: mac-intel-osx-x86_64-10.9
                       OS version: 13.3
                      destination: /usr/local/dials/dials-v3-14-1
                  # of processors: 12
    =========================================================================

Configuring DIALS components...

 -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=- -=-

An error occurred during installation

Excerpt from installation log:
 : /bin/sh: /usr/local/dials/dials-v3-14-1/conda_base/python.app/Contents/MacOS/python: No such file or directory

This led to Traceback (most recent call last):
  File "/usr/local/dials/dials-installer/./bin/install.py", line 69, in reconfigure
    self.reconfigure_as_libtbx_does(log)
  File "/usr/local/dials/dials-installer/./bin/install.py", line 125, in reconfigure_as_libtbx_does
    installer_utils.call(args=args, log=log, verbose=self.options.verbose)
  File "/usr/local/dials/dials-installer/lib/libtbx/auto_build/installer_utils.py", line 55, in call
    raise RuntimeError("Call to '%s' failed with exit code %d" % (args, rc))
RuntimeError: Call to '/usr/local/dials/dials-v3-14-1/conda_base/python.app/Contents/MacOS/python /usr/local/dials/dials-v3-14-1/modules/cctbx_project/libtbx/configure.py --current_working_directory /usr/local/dials/dials-v3-14-1/build --use_conda --skip_phenix_dispatchers dials xia2 iota prime' failed with exit code 127
Please report this installation error to dials-support@lists.sourceforge.net
```

**Comment by ndevenish** [comment]
Hi, from the title of this issue it sounds like you are trying to install the `dials-v3-14-1-linux-x86_64.tar.xz` linux distribution of DIALS? We do have a separate release bundles for mac e.g. [dials-v3-14-1-macosx.tar.gz](https://github.com/dials/dials/releases/download/v3.14.1/dials-v3-14-1-macosx.tar.gz) on the [release](https://github.com/dials/dials/releases/tag/v3.14.1) page; but this is the x86 Mac build instead of Apple Silicon.

DIALS works mostly fine on Silicon macs (several of us develop it this way), and via rosetta still works as far as we are aware, but we don't have native release install bundles yet. The options for getting a pure (non-rosetta-2) installation of DIALS on apple silicon currently are:
- Self-building via downloading and running `bootstrap.py`
- Installation via conda/mamba `conda install -c conda-forge dials`. We just released DIALS 3.14.1, so it hasn't propagated to conda-forge yet, but via conda-forge the differences should be minimal (it seems not to be affected by the issue we hotfixed).
